docs/compatibility_matrix.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility Matrix
+
+This document shows the compatibility of Scalar DB with supported databases.
+
+## Scalar DB Compatibility
```

Comment:
```suggestion
## Compatibility matrix
```

Reply:
Thank you for your suggestion.
Suggestion integrated.